locales/en-global.json5

```
@@ -128,7 +141,12 @@
   today: 'Today',
   more: 'More',
   less: 'Less',
-  search: 'Search',
+  search: {
+    articles: {
+      placeholder: 'Search articles...',
+      ariaLabel: 'Search for articles and documentation',
+    },
+  },
```

unused -> revert to original or delete.
```suggestion
  search: 'Search',
```

---

Done